### PR TITLE
CPE / example cleanup

### PR DESCRIPTION
**Pull request by tsellers-r7**
## Description
This PR includes:
- General fixup after landing PRs
- CPE updates with using today's NIST CPE database

## Motivation and Context
Quality / consistency.

## How Has This Been Tested?
`rspec`

## Checklist:

- [x] I have updated the documentation accordingly (or changes are not required).
- [x] I have added tests to cover my changes (or new tests are not required).
- [x] All new and existing tests passed.

### CODE REVIEW

File: xml/ntp_banners.xml [hunk 1]
```diff
@@ -1032,8 +1032,8 @@
     <param pos="0" name="os.family" value="Data ONTAP"/>
     <param pos="0" name="os.product" value="Data ONTAP"/>
     <param pos="1" name="os.version"/>
-    <param pos="0" name="ow.device" value="NAS"/>
     <param pos="0" name="os.cpe23" value="cpe:/o:netapp:data_ontap:{os.version}"/>
+    <param pos="0" name="ow.device" value="NAS"/>
```
Comment (hdm) [inline xml/ntp_banners.xml:1036]: Typo of ow -> os

File: xml/http_cookies.xml [hunk 2]
```diff
@@ -359,11 +361,12 @@
     <param pos="0" name="service.product" value="Alteon Web Switch"/>
   </fingerprint>
 
-  <fingerprint pattern="^(OBSID)=.*">
+  <fingerprint pattern="^OBSID=.*">
     <description>Observium</description>
-    <param pos="1" name="cookie"/>
+    <param pos="0" name="cookie" value="OBSID"/>
```
Comment (hdm) [inline xml/http_cookies.xml:366]: I am not sure of the point of these, do they provide anything useful to the user?

**Comment by hdm** [comment]
Thanks! Looks like one small typo (also in the original fingerprint). Looks good to me, but I was wondering what the point of the cookie name capture is.

File: xml/http_cookies.xml (hunk 2 above)
Comment (tsellers-r7) [reply xml/http_cookies.xml:366]: I've never used them. I've only kept them there for consistency sake.  I think Jon added them for cases where the cookie name contains a value that might be useful to the consumer.  An example I've seen would be 
`CF_CLIENT_FOO_CORP=2;Path=/;Expires=Sun, 02-Aug-2020 00:54:53 UTC`
`wordpress_89ca499ef62aa0b115202a9a7cf04c10=+; expires=Fri, 02-Aug-2019 01:55:35 GMT; path=/`
`BIGipServer~web~pool_SOMEPOOLNAME=386140170.36895.0000; path=/; Httponly; Secure`

**Pull request merged**